exercises/complex-numbers/src/complex_numbers.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _COMPLEX_NUMBERS_H_
+#define _COMPLEX_NUMBERS_H_
+
+typedef struct Complex {
+    double real;
+    double imag;
+} Complex;
```

Comment:
To be consistent in style, this should be named `complex_t`

Reply:
Will do.